website/frontend/src/components/blocks/Wallet/SelectAccountModal/SelectAccountModal.module.scss

```diff
@@ -0,0 +1,13 @@
+.message {
+   text-align: center;
+ 
+   a {
+     text-decoration: underline;
```

Comment:
I believe it's better to disable `text-decoration` only if `a` is outside of paragraph, in our `index.scss`
```
// underline only if link is a part of paragraph
  &:not(p &) {
    text-decoration: none;
  }
```